pkgs/top-level/python-packages.nix

```diff
@@ -26943,4 +26944,41 @@ in modules // {
       license = licenses.wtfpl;
     };
   };
+
+  # tensorflow is built from a downloaded wheel, because the upstream
+  # project's build system is an arcane beast based on
+  # bazel. Untangling it and building the wheel from source is an open
+  # problem.
```

Comment:
Support for installing wheels was added in #15010. It suffices to add the argument `format="wheel"`.